### PR TITLE
fix intro & status page locale switch

### PR DESCRIPTION
**Pull request by Riscky**

### CODE REVIEW

File: app/assets/javascripts/language.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ I18n.fallbacks = true;
 I18n.defaultLocale = "nl";
 
 // Language switcher
-const urlParams = new URLSearchParams(window.location.search);
```
Comment (Mstiekema) [inline app/assets/javascripts/language.js:10]: Why did you change this? Both are constants

**Comment by Mstiekema** [comment]
"Gezelligheid" is not translated either

File: app/assets/javascripts/language.js (hunk 1 above)
Comment (Riscky) [reply app/assets/javascripts/language.js:10]: it gave errors in the console on switching language

**Comment by Riscky** [comment]
> "Gezelligheid" is not translated either

There is no translation for gezelligheid, people should learn at least this one dutch word ;)

File: app/assets/javascripts/language.js (hunk 1 above)
Comment (Mstiekema) [reply app/assets/javascripts/language.js:10]: Hmm weird, okay!
